src/core/lib/gpr/alloc.h

```diff
@@ -25,4 +25,12 @@
 #define GPR_ROUND_UP_TO_ALIGNMENT_SIZE(x) \
   (((x) + GPR_MAX_ALIGNMENT - 1u) & ~(GPR_MAX_ALIGNMENT - 1u))
 
+#define GPR_ROUND_UP_TO_CACHELINE_SIZE(x) \
+  (((x) + GPR_CACHELINE_SIZE - 1u) & ~(GPR_CACHELINE_SIZE - 1u))
+
+#define GPR_ROUND_UP_TO_SPECIFIED_SIZE(x, align) \
```

Comment:
Maybe rename this to `GPR_ROUND_UP_TO_ALIGNMENT_SIZE()`, and rename the one on line 25 to `GPR_ROUND_UP_TO_MAX_ALIGNMENT_SIZE()`, to better reflect what each one does?

Reply:
Done.